src/main/java/bio/terra/service/search/SearchService.java

```diff
@@ -120,27 +128,41 @@ private SearchIndexModel getIndexSummary(String indexName) {
 
     public SearchIndexModel indexSnapshot(Snapshot snapshot, SearchIndexRequest searchIndexRequest)
         throws InterruptedException {
-        // TODO: add streaming version of this mechanism instead of loading everything into memory
```

Comment:
I removed the TODO since we have a backlog ticket for this https://broadworkbench.atlassian.net/browse/DR-1854

Reply:
Sounds good.